example_queries/breadth.cc

```diff
@@ -243,11 +243,11 @@ void BidiContext::onResponseHeadersInbound() {
     std::string key = b3_trace_id_;
     std::string value;
 
-    std::string x_height = std::to_string(get_out_degree(target, mapping->at("x")));
+    std::string x_breadth = std::to_string(get_out_degree(target, mapping->at("x")));
```

Comment:
This is just a rename for clarification, right?

Reply:
Yes, exactly.